### DISCUSSION TITLE
Unable to install Cypress as npm was not installed properly

### DISCUSSION THREAD

**Discussion by suneelKalyanAmancha** [npm/rfcs#623] [General]
Hello TECHies,
This is Suneel, while Installing Cypress I'm getting some errors as npm is not installed properly. I'd like to uninstall and reinstall npm again. Here I'm attaching screenshots of the error. Please help me out.
![cmd error](https://user-images.githubusercontent.com/110740153/183262170-cdfcc03c-be6e-4ec6-98d5-bdb27a92195c.jpg)
![VS Error](https://user-images.githubusercontent.com/110740153/183262172-fb50f0f3-9c83-42c6-83a5-fb902f779454.jpg)

Thanks,
SUNIL.

**Comment by gvoze32** [comment]
You've misspelled `node_modules` in your command `node_module\.bin\cypress open` with node_module instead of node_modules.

**Reply by suneelKalyanAmancha** [reply]
No, I have copied the code from Cypress official site.

**Reply by ljharb** [reply]
ok, then _they_ misspelled it. it's still not correct.

**Reply by IgorKowalczyk** [reply]
@suneelKalyanAmancha https://docs.cypress.io/guides/getting-started/opening-the-app#cypress-open

There is no typo in the documentation. You are just misspelled the command
